lib/commands/plugin-test.sh

```diff
@@ -39,7 +39,9 @@ plugin_test_command() {
     fi
 
     TEST_DIR=$(mktemp -dt asdf.XXXX)
-    git clone "$ASDF_DIR/.git" "$TEST_DIR"
+    cp -R "$ASDF_DIR/bin" "$TEST_DIR"
+    cp -R "$ASDF_DIR/lib" "$TEST_DIR"
+    cp -R "$ASDF_DIR/asdf.sh" "$TEST_DIR"
```

Comment:
@Stratus3D This seems to me to be enough to copy over, at least in my rudimentary testing of a couple things, did I miss anything?

Reply:
Looks good to me.